### ISSUE TITLE
Page has no heading

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#376]
![screen shot 2016-07-06 at 15 05 43](https://cloud.githubusercontent.com/assets/13200338/16620755/8dc80fd4-438b-11e6-91fd-3d467735d803.png)

N.b. This is only a warning and so will not fail WCAG 2.0 AA standard 
N.b.b. This may have no relevance as may be unique to demo app

**Comment by VitoAlbano** [comment]
@hashraf88 this is an alert do we need to fix it to match the "AA"?

**Comment by hashraf88** [comment]
I think we are ok as firstly it is only a warning and this is specific to the demo shell.
Will close and we can revisit if we need to.

**Comment by VitoAlbano** [comment]
ok i close the issue then.
Thanks
